tests/components/switchbot/test_sensor.py

```diff
@@ -151,3 +152,49 @@ async def test_relay_switch_1pm_power_sensor(hass: HomeAssistant) -> None:
 
     assert await hass.config_entries.async_unload(entry.entry_id)
     await hass.async_block_till_done()
+
+
+@pytest.mark.usefixtures("entity_registry_enabled_by_default")
+async def test_leak_sensor(hass: HomeAssistant) -> None:
+    """Test setting up the leak detector."""
+    await async_setup_component(hass, DOMAIN, {})
+    inject_bluetooth_service_info(hass, LEAK_SERVICE_INFO)
+
+    with patch(
+        "switchbot.SwitchbotRelaySwitch.update",
+        return_value=None,
+    ):
```

Comment:
This shouldn't needed since its not a relay switch

Reply:
Thanks for catching that! I removed it.